Makefile

```diff
@@ -104,14 +106,33 @@ clean: ## Remove all generated files
 	rm -f bazel-*
 
 .PHONY: docker-build
-docker-build: generate fmt vet manifests ## Build the docker image
-	docker build . -t ${IMG}
+docker-build: generate fmt vet manifests ## Build the docker image for controller-manager
+	docker build . -t ${CONTROLLER_IMG}
 	@echo "updating kustomize image patch file for manager resource"
-	sed -i.tmp -e 's@image: .*@image: '"${IMG}"'@' ./config/default/manager_image_patch.yaml
+	sed -i.tmp -e 's@image: .*@image: '"${CONTROLLER_IMG}"'@' ./config/default//manager_image_patch.yaml
```

Comment:
It'd be great to remove `sed` as part of this PR and fixing #661. We could use `envsubst` or make it work across Mac/Linux.

Reply:
@vincepri i did some research on `envsubst`, it is not available on many containers by default, i would hesitate to introduce a new dependency for a simple function, on the other hand, the current way of using `sed` is working in the same way cross MacOS and Linux, #661 is already fixed in my opinion.